src/main/app/src/assets/i18n/nl.json

```diff
@@ -127,6 +127,8 @@
   "doel": "Doel",
   "door": "Door",
   "verzender": "Afzender",
+  "verzender.-geen-": "- Geen E-Mail adres -",
```

Comment:
https://taaladvies.net/e-mail-hoofdletter/#:~:text=Bij%20e%2Dmail%20is%20er,de%20vorm%20van%20een%20object.
https://taaladvies.net/email-of-e-mail/
https://zakelijk-schrijfadvies.nl/e-mail-of-email/

 :smiley: